latest/index.bs

```diff
@@ -102,6 +102,13 @@ is represented here as it would appear locally but could equally
 be stored on a web server to be accessed via HTTP or in object storage
 like S3 or GCS.
 
+OME-Zarr is an implementation of the OME-NGFF specification using the Zarr
+format. Arrays MUST be defined and stored in a hierarchical organization as
```

Comment:
All for defining this, but it _might_ be that it would already "just work" with some implementations (e.g. a pre-release of zarr-python). Very similar to the `dimension_separator`. I imagine to drop the restriction will be a matter of testing the implementations that we consider MUST HAVEs, and if they pass, setting their minimal versions and dropping this clause.

Reply:
All agreed, I confess I have little knowledge of the compatibility with the upcoming spec. I went for the conservative approach of starting with a restrictive definition of the supported Zarr layout. This requirement can certainly be reviewed and relaxed. Maybe worth capturing as an issue?

Reply:
:+1: for restrictive-first.

Reply:
I would also vote for restrictive first here.